crates/sui-graphql-rpc/src/types/unchanged_shared_object.rs

```diff
@@ -39,6 +40,12 @@ pub(crate) struct SharedObjectDelete {
     mutable: bool,
 }
 
+#[derive(SimpleObject)]
+pub(crate) struct SharedObjectCancelled {
```

Comment:
Can you add a doc comment here explaining how this kind of input maniefsts? It will get ported into the doc comments in the schema.

Reply:
Good point! Doc added.